packages/util-dev/src/stubLogger.ts

```diff
@@ -1,4 +1,5 @@
 import { Logger } from 'ts-log';
+import _ from 'jest';
```

Comment:
`util-dev` should only be used in `package/test` project and you shouldn't need to build this
```suggestion
import 'jest';
```

Reply:
There is no distinction between build time and test time dependencies and various devDependencies are effectively build time dependencies, such as `node-run-all` or `shx` or `typescript`, for some examples. Hence, we need all dev dependencies in lack of a more granular distinction.

Reply:
Right, but `util-dev` should only be referenced by `[package]/test/tsconfig.json` and you should only need to build using `[package]/src/tsconfig.json` - so this `stubLogger.ts` file shouldn't be used by `tsc`. I'm wondering if you could reduce the number of packages to build by only building `cardano-services` and it's dependencies as [referenced](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/cardano-services/src/tsconfig.json#L7) by it's tsconfig.json.